automatic/minishift/minishift.nuspec

```diff
@@ -3,10 +3,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>minishift</id>
-    <version>1.34.3</version>
+    <version>1.34.3.20210527</version>
```

Comment:
As this package uses `AU` to update and push any changes, there is no need to update the version number manually with a package fix.

Forcing a push will create a package fix automatically.